web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/ProgressBar.tsx

```diff
@@ -108,29 +114,19 @@ const StyledLine = styled(Box)`
 
 function PhaseIcon({ status }: { status: AttemptStatus }): JSX.Element {
   if (status === 'success') {
-    return <Check size="small" color="white" />;
+    return <icons.Check size="small" color="white" />;
   }
 
   if (status === 'error') {
-    return <Warning size="small" color="white" />;
+    return <icons.Warning size="small" color="white" />;
   }
 
   if (status === 'processing') {
     return (
-      <Refresh
-        size="extraLarge"
-        color="success"
-        css={`
-          animation: anim-rotate 1.5s infinite linear;
-          @keyframes anim-rotate {
-            0% {
-              transform: rotate(0);
-            }
-            100% {
-              transform: rotate(360deg);
-            }
-        `}
-      />
+      <>
+        <Spinner />
+        <icons.Restore size="small" color="buttons.text" />
```

Comment:
TBH the restore icon doesn't make much sense to me (because we are not restoring anything 😅), but I see that we want to keep things consistent.

Reply:
Yeah, I agree about the icon but I wanted to make it look the same between Cloud and here.